homeassistant/components/notify/pushover.py

```diff
@@ -51,7 +51,21 @@ def send_message(self, message="", **kwargs):
         """Send a message to a user."""
         from pushover import RequestError
 
+        data = kwargs.get(ATTR_DATA)
```

Comment:
Thank you @balloob, I've committed the fix.

I'll be able to run the local tests in a few days when I figure it all out.